test/transcoding/PipeStorage.ll

```diff
@@ -11,9 +11,6 @@
 ; CHECK-LLVM-DAG: [[CREATOR_NAME:[^ ]+]] = linkonce_odr addrspace(1) global %spirv.ConstantPipeStorage { i32 16, i32 16, i32 1 }, align 4
 ; CHECK-LLVM-DAG: @mygpipe = addrspace(1) global %"[[CL_PIPE_STORAGE_NAME]]" { %spirv.PipeStorage addrspace(1)* bitcast (%spirv.ConstantPipeStorage addrspace(1)* [[CREATOR_NAME]] to %spirv.PipeStorage addrspace(1)*) }, align 4
 
-; check for magic number followed by version 1.1
```

Comment:
Why your changes affected this test?

Reply:
Because I'm setting spirv1.4 version (if translator allowed to use it) if there are global variables with a storage class != input / output. So this test fails as it expexts version 1.1.